todo_app/data/session_items.py

```diff
@@ -9,90 +10,75 @@
 
 # Returns a list of todo items
 def get_items():
-    url = "https://api.trello.com/1/lists/" + app.config['LIST']+ "/cards"
-    query = {
-		'key': app.config['TRELLO_KEY'],
-        'token': app.config['TOKEN'],
-    	}
-	
-    response = requests.request('GET', url, params=query)
-    x=response.json()
-    
-    items = [ ]
-    for i in x:
-        items.append(Item(i['id'], 'ToDo', i['name'] ))
-	
-    return items
-
-# Returns a list of completed items
+    todo_items = []
+    mongo_url = app.config['MONGO_URL']
+    mongo_db = app.config['MONGO_DB']
+
+    client = pymongo.MongoClient(mongo_url)
+    db = client[mongo_db]
+    items = db.items
+    todo_items = items.find({"status": "ToDo"})
+
+    return todo_items
```

Comment:
Now that you've stopped using your Item class, you're no longer guaranteeing that the rest of the app sees the same interface as before; and that is actually an issue as it's broken your "complete" method. The immediate reason is that the front end expects an "id" property but this now returns an "_id" property instead, but a better solution than changing the template is to still use our Item class:
```suggestion
    todo_items = items.find({"status": "ToDo"})
    
    items = []
    for item in todo_items:
        items.append(Item(item['_id'], item['status'], item['title']))
    return items
```

(This is also true for the `get_completeitems` function below)